platforms/software/maven/src/integTest/groovy/org/gradle/api/publish/maven/MavenPublishHttpIntegTest.groovy

```diff
@@ -360,7 +361,7 @@ class MavenPublishHttpIntegTest extends AbstractMavenPublishIntegTest {
         then:
         module.assertPublishedAsJavaModule()
         if (GradleContextualExecuter.isConfigCache()) {
-            postBuildOutputContains("Configuration cache entry discarded")
+            result.assertHasPostBuildOutput("Configuration cache disabled because incompatible task was found.")
```

Comment:
💅 `postBuildOutputContains` is literally an assertion on having a result + `result.assertHasPostBuildOutput`. I'd use old version as it more concise.